### ISSUE TITLE
Add better tests

### ISSUE THREAD

**Issue by LetsMelon** [LetsMelon/addIdToArray#1]

**Comment by LetsMelon** [comment]
Find duplicates with [pmd](https://github.com/pmd/pmd)
Example command:
```sh
./run.sh cpd --minimum-tokens [INSER_NUMBER] --files [SOMETHING/addIdToArray/index.test.js]  --language javascript
```

**Comment by LetsMelon** [comment]
Fixed with commit [c1b6e452a433fd569f3a6cb0bc90dad13b46fa44](https://github.com/LetsMelon/addIdToArray/commit/c1b6e452a433fd569f3a6cb0bc90dad13b46fa44)?
